doc/source/admin/galaxy_options.rst

```diff
@@ -1381,9 +1381,7 @@
 ~~~~~~~~~~~~~~~~
 
 :Description:
-    This flag enables an AWS cost estimate for every job based on their runtime matrices.
-    CPU, RAM and runtime usage is mapped against AWS pricing table.
-    Please note, that those numbers are only estimates.
+    Enable AWS estimate.
```

Comment:
I think the proper fix here would be to copy the long option description to `lib/galaxy/webapps/galaxy/config_schema.yml`

Reply:
Done. It appears that text was only in `galaxy_options.rst`, so it got overwritten on rebuild. I assumed that was an updated/shortened description. Thanks!